src/mame/drivers/altos5.cpp

```diff
@@ -303,6 +308,54 @@ WRITE8_MEMBER( altos5_state::port09_w )
 	setup_banks(2);
 }
 
+
+/***********************************************************
+
+    Quickload
+
+    This loads a .COM file to address 0x100 then jumps
+    there. Sometimes .COM has been renamed to .CPM to
+    prevent windows going ballistic. These can be loaded
+    as well.
+
+************************************************************/
+
+QUICKLOAD_LOAD_MEMBER( altos5_state, altos5 )
+{
+	address_space& prog_space = m_maincpu->space(AS_PROGRAM);
+	
+	if (quickload_size >= 0xfd00)
+		return image_init_result::FAIL;
+	
+	setup_banks(2);
+
+	/* Avoid loading a program if CP/M-80 is not in memory */
+	if ((prog_space.read_byte(0) != 0xc3) || (prog_space.read_byte(5) != 0xc3)) {
```

Comment:
You still have mixed Allman and K&R formatting within function.  For example this `if` statement has the opening brace of the controlled block (K&R style) while the `for` loop below has the opening brace of the controlled statement on the following line (Allman style).

In each case, you should look at the rest of the file, and follow the same convention, whether it's K&R or Allman.  You should not mix Allman and K&R within a file, and particularly not within a function.